### ISSUE TITLE
DetailsList column headers are skipped when using Narrator's table navigation

### ISSUE THREAD

**Issue by islegare** [microsoft/fluentui#19469]
### Environment Information

- **Package version(s)**: 8.29.0
- **Browser and OS versions**: Chrome Windows

### Describe the issue:

When trying to navigate column headers using Narrators table hotkeys (Ctrl + Alt + Right/Left arrow). It skips headers, and I'm unable to go to a previous column header.

### Please provide a reproduction of the issue in a codepen:
Can be reproduced in the default codepen from https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist

#### Actual behavior:

Column headers are skipped when using Narrator's table navigation hotkeys

#### Expected behavior:

Each Column header is hit when using Narrator's table navigation hotkeys

**Comment by gouttierre** [comment]
@islegare - Thanks for filing this issue with us. To set expectation our team will have a look at the issue but only once capacity allows.

@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by smhigley** [comment]
This was a bug specifically with Edge 92 and and off-by-one index issue with mapping `aria-rowindex` and `aria-colindex` to UIA, not specific to Fluent. A bug was filed, and it was fixed. You can see the correct behavior in any Edge version above 92 :).

**Comment by islegare** [comment]
Thanks for the info!
